celery/loaders/base.py

```
@@ -255,21 +266,33 @@ def find_related_module(package, related_name):
     """Find module in package."""
     # Django 1.7 allows for speciying a class name in INSTALLED_APPS.
     # (Issue #2248).
+    print("EDX-CELERY-FindingTasksModule: Finding tasks module for %s" % package)
+    logger.info("EDX-CELERY-FindingTasksModule: Finding tasks module for %s", package)
     try:
         module = importlib.import_module(package)
         if not related_name and module:
             return module
     except ImportError:
-        package, _, _ = package.rpartition('.')
-        if not package:
+        new_package, _, _ = package.rpartition('.')
+        if not new_package:
+            logger.info('EDX-CELERY-PackageNotFound: No package found in %s', package)
+            print('EDX-CELERY-PackageNotFound: No package found in %s' % package)
             raise
-
+        package = new_package
     module_name = '{0}.{1}'.format(package, related_name)
-
     try:
-        return importlib.import_module(module_name)
+        import_module = importlib.import_module(module_name)
+        print("EDX-CELERY-TaskModuleImported: %s module is imported" %
+              import_module.__name__)
+        logger.info("EDX-CELERY-TaskModuleImported: %s module is imported",
+                    import_module.__name__)
+        return import_module
     except ImportError as e:
         import_exc_name = getattr(e, 'name', module_name)
         if import_exc_name is not None and import_exc_name != module_name:
+            logger.exception('EDX-CELERY-ErrorInImport: Found some error in %s', package)
+            print('EDX-CELERY-ErrorInImport: Found some error in %s' % package)
```

Maybe log/print the exception itself also, in case something silently swallows it later?